### ISSUE TITLE
Naming of PageModelBase

### ISSUE THREAD

**Issue by Daeymon** [mikoskinen/Blazor.Page#1]
The naming of the context class is a little misleading and may cause conflicts in pre-existing projects with Razor pages if the developer has created a base class between all their PageModel classes (usually named PageModelBase).

Might be an idea to give yours a different name so its clear its not related to a PageModel class and won't cause conflicts in projects which already have a PageModelBase for Razor pages.

**Comment by mikoskinen** [comment]
Yep, that's true. The property and class names need a little rethink to make things more clear . Now they are a mismatch of Razor Pages (PageModelBase) and WPF/Silverlight (DataContext).

I've been thinking of borrowing some naming conventions from Caliburn.Micro, where view models usually inherit Screen.
